lib/browser/rpc-server.js

```diff
@@ -263,10 +263,14 @@ const callFunction = function (event, contextId, func, caller, args) {
 const handleRemoteCommand = function (channel, handler) {
   ipcMain.on(channel, (event, contextId, ...args) => {
     let returnValue
-    try {
-      returnValue = handler(event, contextId, ...args)
-    } catch (error) {
-      returnValue = exceptionToMeta(event.sender, contextId, error)
+    if (event.sender._isRemoteModuleEnabled()) {
+      try {
+        returnValue = handler(event, contextId, ...args)
+      } catch (error) {
+        returnValue = exceptionToMeta(event.sender, contextId, error)
+      }
+    } else {
+      returnValue = exceptionToMeta(event.sender, contextId, new Error('remote is disabled'))
```

Comment:
Can we ensure that `returnValue` here is just null on the renderer side and a console message is logged instead of an error being thrown.  I don't want to somehow leak the `event` or `event.sender` here.  The safest course is to not return anything from `remote` methods when it is disabled.

Reply:
done